src/components/tables/DataTable/plugins/useRowSelectColumn.tsx

```diff
@@ -32,7 +32,11 @@ export const useRowSelectColumn = <D extends object>(hooks: ReactTable.Hooks<D>)
                   <Checkbox
                     aria-label="Select all rows"
                     checked={checked}
-                    onChange={onChange}
+                    onClick={(e) => e.stopPropagation()}
```

Comment:
Some quick code style requests:

- Name the event param `event`, for consistency
- Don't return the result of `stopPropagation` to the `onClick`, wrap the body of the function in `{}`:

```
onClick={event => { event.stopPropagation(); }}
```